config/file_config.go

```diff
@@ -239,6 +240,13 @@ type SmartWrapperOptions struct {
 	ReaperRunInterval Duration `yaml:"ReaperRunInterval" default:"1h"`
 }
 
+func (c CollectionConfig) GetShutdownDelay() time.Duration {
```

Comment:
Given that we have `GetCollectionConfig` and that you're using it, I'm leaning toward getting rid of the specific field accessors in favor of just accessing it. It has a default and we can put a "nonzero" validator on it, so we can be sure it's never 0. That does mean that tests have to be sure to set it, but I think that's acceptable in favor of getting rid of a few dozen accessors. 

(This is a request to get rid of this, not a demand.)

Reply:
I like the idea. We can make this change in another PR